scripts/coala_patch.sh

```diff
@@ -8,3 +8,5 @@ git commit --message "Coala auto-patch for Travis CI Build:$TRAVIS_BUILD_NUMBER
 git remote add origin-pages https://coala-autofix-bot:$COALA_FIXER@github.com/fossasia/gci16.fossasia.org.git
 git pull --rebase origin-pages coala-fix
 git push --quiet --set-upstream origin-pages coala-fix
+response = $(curl --write-out '%{http_code}' -output /dev/null -X POST--H "Authorization: token $GITHUB_TOKEN" --data '{"title":"coala-fixes", "body":"Automated Coala fixes", "head":"coala-autofix", "base":"master"}' https://api.github.com/repos/fossasia/gci16.fossasia.org/pulls)
+echo $response
```

Comment:
Lines 11 and 12 don't work. 

Just write out the returned JSON, no need to be fancy.

`curl -X POST -H "Authorization: token $GITHUB_TOKEN" --data '{"title":"coala-fixes for build $TRAVIS_BUILD_NUMBER", "body":"Automated Coala fixes:$TRAVIS_BUILD_NUMBER", "head":"coala-autofix", "base":"master"}' https://api.github.com/repos/fossasia/gci16.fossasia.org/pulls`